backend/django/core/utils/util.py

```diff
@@ -555,7 +558,7 @@ def get_excluded_data(project):
             temp.update(MetaDataSerializer(metadata).data)
 
         data_labels = DataLabel.objects.filter(data=d.data)
-        for i in range(data_labels.count()):
+        for i, label in enumerate(data_labels):
             temp[f"label_{i}"] = data_labels[i].label.name
             temp[f"label_{i}_reason"] = data_labels[i].label_reason
```

Comment:
since you used enumerate you now have access to the item that is in the data_labels list at index i.  So instead of `data_labels[i].label.name` you should just be able to use `label.name`